README.md

```diff
@@ -1,4 +1,30 @@
 # Kotlin Wallet SDK
 
 Kotlin Wallet SDK is a library that allows developers to build wallet applications on the Stellar network faster. It
-utilizes [Java Stellar SDK](https://github.com/stellar/java-stellar-sdk) to communicate with a Stellar Horizon network.
\ No newline at end of file
+utilizes [Java Stellar SDK](https://github.com/stellar/java-stellar-sdk) to communicate with a Stellar Horizon server.
+
+## Wallet
+
+`class Wallet(horizonUrl: String, networkPassphrase: String)`
+
+Example
+
+```kotlin
+import org.stellar.sdk.Network
+
+// Using Horizon testnet configuration
+val wallet = Wallet("https://horizon-testnet.stellar.org", Network.TESTNET.toString())
```

Comment:
You may use static imports to further simplify. 
```suggestion
val wallet = Wallet("https://horizon-testnet.stellar.org", TESTNET.toString())
```

IntelliJ has auto code complete for static imports. ([Stack Overflow](https://stackoverflow.com/questions/4387134/intellij-static-import-completion))